src/frontend/src/tests/sol/schedulers/sol-wallet.scheduler.spec.ts

```diff
@@ -5,12 +5,13 @@ import * as solanaApi from '$sol/api/solana.api';
 import { SolWalletScheduler } from '$sol/schedulers/sol-wallet.scheduler';
 import { SolanaNetworks } from '$sol/types/network';
 import { mockIdentity } from '$tests/mocks/identity.mock';
+import { lamports, type Lamports } from '@solana/rpc-types';
 import { type MockInstance } from 'vitest';
 
 describe('sol-wallet.scheduler', () => {
 	let spyLoadBalance: MockInstance;
 
-	const mockBalance = 100n;
+	const mockBalance: Lamports = lamports(100n);
```

Comment:
```
// in this test
const mockBalance: Lamports = lamports(100n);

// vs in the other test of this PR
const mockBalance = lamports(1000n);
```

Do we need `: Lamports` or it can be omitted?